src/build/arg/arg_value.rs

```diff
@@ -0,0 +1,77 @@
+use std::fmt::Display;
+
+/// TODO
+#[derive(Debug, Default, Clone)]
+pub struct ArgValue<'help> {
+    pub(crate) value: &'help str,
+    pub(crate) about: Option<&'help str>,
+    pub(crate) hidden: bool,
+}
+
+impl Display for ArgValue<'_> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if self.value.contains(char::is_whitespace) {
+            write!(f, "{:?}", self.value)
+        } else {
+            write!(f, "{}", self.value)
+        }
+    }
+}
+
+impl<'help> From<&'help str> for ArgValue<'help> {
+    fn from(s: &'help str) -> Self {
+        Self::new(s)
+    }
+}
+
+impl<'help> From<&&'help str> for ArgValue<'help> {
+    fn from(s: &&'help str) -> Self {
+        Self::new(*s)
+    }
+}
+
+/// Getters
+impl<'help> ArgValue<'help> {
+    /// TODO
+    #[inline]
+    pub fn get_value(&self) -> &str {
+        self.value
+    }
+
+    /// TODO
+    #[inline]
+    pub fn get_about(&self) -> Option<&str> {
+        self.about
+    }
+
+    /// TODO
+    #[inline]
+    pub fn get_hidden(&self) -> bool {
```

Comment:
Shouldn't this be `is_hidden`?

Reply:
makes sense, would have been my instinct as well, just saw get_boolean in other parts of clap

Reply:
Probably things for us to audit and adjust